src/main/groovy/org/gradle/android/workarounds/room/KaptWorkaround.groovy

```diff
@@ -143,4 +147,11 @@ class KaptWorkaround extends AnnotationProcessorWorkaround<KaptRoomSchemaLocatio
             throw new RuntimeException("Field '${fieldName}' not found")
         }
     }
+
+    // Before adding the ArgumentProvider, we need to verify that the room-processor is applied in the kapt configuration
+    private boolean kaptContainsRoom() {
+        return project.configurations.named("kapt").get().dependencies.name
```

Comment:
@bigdaz can you confirm this is correct? 

At this point, we are checking the configurations after the android variants have been resolved.

Reply:
thanks @bigdaz  +1000